### ISSUE TITLE
Suggestion: clarify units in charts

### ISSUE THREAD

**Issue by cgrigis** [mlbench/mlbench-dashboard#27]
I think the following could clarify the reading of the dashboard graphs:

Node charts:
* Unit for Memory: `Mb` → `MB`
* Unit for Network in / out: `Mb/s` → `MB/s` (assuming it is megabytes/sec?)
* When focusing on a node, there is no longer any unit displayed.

In general:
* Time units seem to be wall clock UTC, in 12h mode. A note indicating this somewhere could be useful, and maybe displaying in 24h mode?
* Do the various metrics for runs have units?

**Comment by Panaetius** [comment]
Yes it's UTC wallclock on the server (to keep things consistent). But it'd be nice to convert it to local wall clock in the browser
